heartbeat/portblock

```diff
@@ -372,43 +556,136 @@ SayInactive()
   ocf_log debug "$CMD DROP rule [$*] is inactive"
 }
 
-#IptablesStatus  {udp|tcp} portno,portno ip {in|out|both} {block|unblock}
-IptablesStatus() {
+SayRulesMissing()
+{
+  ocf_log err "$CMD DROP rule [$*] is missing - returning OCF_NOT_RUNNING"
+}
+
+#PortStatus  {udp|tcp} portno,portno ip {in|out|both} {block|unblock}
+PortStatus() {
   local rc
   rc=$OCF_ERR_GENERIC
   is_active=0
+
+  ocf_log info "PortStatus: Starting monitor check for $1 ports $2 on $3 direction $4 action $5"
+  ocf_log info "PortStatus: FIREWALL=$FIREWALL"
+
   if [ "$4" = "in" ] || [ "$4" = "both" ]; then
+    ocf_log info "PortStatus: Calling chain_isactive for INPUT chain"
```

Comment:
info might be a bit too verbose here.
maybe info for actual results and some debug level for showing how that was gathered.